repose-aggregator/functional-tests/spock-functional-test/src/test/configs/features/core/intrafilterlogging/header-normalization.cfg.xml

```diff
@@ -22,14 +22,6 @@
             </whitelist>
         </target>
 
-        <!-- applies to all POST and PUT requests -->
-        <target http-methods="POST PUT">
-            <whitelist id="modification">
-                <header id="X-THIRD-Filter"/>
-                <header id="x-auth-token"/>
-            </whitelist>
-        </target>
-
```

Comment:
The original intra-filter logging functional test includes all of the filters, and for the new functional test, I removed some of the filters that would have required additional effort to get the request to successfully complete.  I actually would have preferred testing as many filters as possible to ensure my updates worked with how every filter handles the new wrappers.  During development, I ran into a problem with how the rate limiting filter messes with the streams which fed my paranoia.